core/lib/tasks/migrations/create_ledger_entries_for_store_credits.rake

```diff
@@ -0,0 +1,20 @@
+namespace :solidus do
+  namespace :migrations do
+    namespace :create_ledger_entries_for_store_credits do
+      task up: :environment do
+        print 'Creating opening ledger entries for store credits...'
```

Comment:
I would rather not print anything in our migrations (since this will show up in each `rake test_app`, `rake sandbox`, and `rspec` run). Though I believe `say` in an actual migration will be silent if `VERBOSE=false` or something.